src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -87,6 +87,7 @@ public Command parseCommand(String userInput) throws ParseException {
             return new FindCommandParser().parse(arguments);
 
         case ListCommand.COMMAND_WORD:
+            new TextToSpeechParser().parse("Listing");
```

Comment:
Perhaps in future iterations, we could have a public constant variable in each command for the `TextToSpeechParser` to parse.

Reply:
I don't think we need a parser for this. We can just instantiate a `TextToSpeech("Listing")` directly.

I agree with @nicholaschuayunzhi on having constants in each command. If you want to go further, we can skip the TTS and embed the sound files directly. This is better as it removes the roundtrip latency required by the API call, but we're then stuck with pre-defined strings.